examples/unit-example/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "unit-example",
+  "private": true,
+  "version": "0.1.0-alpha.10",
+  "description": "Example usage of the Unit-compatible functionality in the sandbar client library",
+  "scripts": {
+    "test": "jest --passWithNoTests",
+    "fmt": "prettier",
+    "build": "tsc --build tsconfig.build.json",
+    "clean": "tsc --build tsconfig.build.json --clean",
+    "dev": "./src/index.ts"
+  },
+  "main": "index.js",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "commander": "^9.0.0",
+    "humanize-duration": "^3.27.1",
+    "sandbar": "workspace:^0.1.0-alpha.10"
```

this is why jest is failing.

```suggestion
    "sandbar": "^0.1.0-alpha.10"
```

it's an outstanding issue tracked here https://github.com/lerna/lerna/issues/2564 ; lerna has been unmaintained for some time (until about 3 weeks ago when the maintainer posted that he's handing the project over to someone else).

anyway, `pnpm` updates the versions for us and uses the local copy at local dev time, so it's okay not to use the workspace version specifier.